buildSrc/build.gradle.kts

```diff
@@ -38,24 +38,22 @@ repositories {
 dependencies {
   implementation(gradleApi())
   implementation(localGroovy())
-  implementation("com.github.jengelman.gradle.plugins:shadow:6.1.0-log4jfix-SNAPSHOT")
-  implementation("gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.5.0")
+  implementation("gradle.plugin.com.github.johnrengelman:shadow:7.1.1")
```

Comment:
You can also delete `jarlibs` directory and remove L34 related to that above.

Reply:
lets do that in a different PR? this one seems to be passing, I'd rather get it merged in its current state. If I need to do other edits, I'll do that as well.

Reply:
Done.